modules/loadbalancer/.header.md

```diff
@@ -32,11 +32,12 @@ module "lbi" {
   name                = "private-lb"
   region              = "West Europe"
   resource_group_name = "existing-rg"
+  backend_name = "vmseries_backend"
 
   frontend_ips = {
     ha = {
       name               = "HA"
-      subnet_id          = "/subscription/xxxx/......."
+      subnet_id          = azurerm_subnet.this.id
```

Comment:
I'm not sure about this change. I don't think we should reference external resource attributes in the code snippets. It's better to leave a string-type placeholder there. Neither of these options will work out of the box but for using the module independently, the latter is more intuitive.